Microsoft.Toolkit.Uwp.UI.Controls/TileControl/TileControl.cs

```diff
@@ -453,90 +457,77 @@ private void CreateModuloExpression(ScrollViewer scrollViewer, double imageWidth
             var compositor = _containerVisual.Compositor;
 
             // Setup the expression
-            ExpressionNode expressionX = null;
-            ExpressionNode expressionY = null;
-            ExpressionNode expressionXVal;
-            ExpressionNode expressionYVal;
+            var expressionX = compositor.CreateExpressionAnimation();
+            var expressionY = compositor.CreateExpressionAnimation();
 
             var propertySetModulo = compositor.CreatePropertySet();
-            propertySetModulo.InsertScalar(imageWidthParam, (float)imageWidth);
+            propertySetModulo.InsertScalar(imageHeightParam, (float)imageWidth);
```

Comment:
There's are flipped? Was that a bug before or is this a mistake?

Reply:
O nope, an involuntary change as a result of referencing code from a different commit. Fixing now

Reply:
Fixed fcd9a3b